src/inc/fixuppointer.h

```diff
@@ -94,6 +122,7 @@ class RelativePointer
         return dac_cast<DPTR(RelativePointer<PTR_TYPE>)>(base)->GetValueMaybeNull(base);
     }
 
+#ifndef DACCESS_COMPILE
```

Comment:
@jkotas, Thanks for review

SetValue shouldn't be used in DAC builds, so we'll remove based versions of SetValue completely and rewrite SetValue without base for non DAC builds. This change relates to RelativePointer implementation in general, and not to this pull request specifically. So we'll open another pull request for this changes.